packages/common/amplify_db_common/example/pubspec.yaml

```diff
@@ -1,16 +1,21 @@
 name: amplify_db_common_example
 description: Demonstrates how to use the amplify_db_common plugin.
-publish_to: 'none'
+publish_to: "none"
 
 environment:
-  sdk: '>=2.18.0 <4.0.0'
+  sdk: ">=2.18.0 <4.0.0"
 
 dependencies:
   amplify_db_common: ">=0.1.2+3 <0.2.0"
   drift: ">=2.4.0 <2.6.0"
   flutter:
     sdk: flutter
 
+dependency_overrides:
```

Comment:
Could we add comments here explaining why we have to use overrides instead of regular dependencies for these imports?

Reply:
Added 👍